### DISCUSSION TITLE
Are there release notes for new versions?

### DISCUSSION THREAD

**Discussion by jmoleary** [dotnet-state-machine/stateless#580] [General]
I tend to update my NuGet package of Stateless every time there is a new version on nuget.org.  But I don't know what's in the releases.  Are there release notes?  When I click the "Releases" link from the github page it just tells me "there are no releases here"

**Comment by Fieslix** [comment]
Probably looking for [CHANGELOG.md](https://github.com/dotnet-state-machine/stateless/blob/dev/CHANGELOG.md)

**Reply by mclift** [reply]
Beat me to it by a whisker!

**Comment by mclift** [comment]
Hi,

Yes, the release notes are in [CHANGELOG.md](https://github.com/dotnet-state-machine/stateless/blob/dev/CHANGELOG.md).

GitHub release tagging hasn't been used for this project in past releases, and while I keep thinking we should update the process, so far I've just replicated what's gone before as it's worked well enough so far.

HTH!

**Comment by mclift** [comment]
Gonna mark this as resolved thanks to the answer provided by @Fieslix
